test/dotnet-bootstrapper.Tests/EndToEndTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Bootstrapper.Tests;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Bootstrapper
+{
+    public class EndToEndBootstrapperTest
+    {
+        private readonly static string artifactsDirectory = Path.GetFullPath(Path.Combine(Directory.GetCurrentDirectory(), "..", "..", "..", "..", "..", "artifacts"));
+        private readonly static string executablePath = Path.Combine(artifactsDirectory, "bin", "dotnet-bootstrapper", TestUtilities.GetConfiguration(), TestUtilities.GetTargetFramework(), TestUtilities.GetRuntimeIdentifier(),
+            RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "dotnet-bootstrapper.exe" : "dotnet-bootstrapper");
```

Comment:
I think it would be good to be a bit more resilient to changes in the folder layout.  Probably a good strategy is to find the repo root by looking for a `.git` folder and then use a relative path from there.

See `TestContext.GetRepoRoot` in dotnet/sdk for a sample of how to do that.  There's also a bunch more code in there for finding different folders, we probably don't need most of that complexity for this repo.